test/metabase/models/data_permissions_test.clj

```diff
@@ -151,6 +165,7 @@
                  :model/Database                   {database-id :id} {}
                  :model/Table                      {table-id-1 :id}  {:db_id database-id}
                  :model/Table                      {table-id-2 :id}  {:db_id database-id}]
+    ;; for cache
```

Comment:
don't understand this comment

Reply:
Doh, sorry - I started writing a comment, then moved things around and forgot to delete it.